transformations.go

```diff
@@ -264,8 +269,13 @@ func (f *combinerTransformation) UnmarshalJSON(data []byte) error {
 }
 
 type Reducer interface {
-	InitialValue() interface{}
-	Reduce(ctx Context, prev interface{}, cur *lrdd.Row) (next interface{}, err error)
+	InitialValue() MarshalerUnmarshaler
+	Reduce(ctx Context, prev MarshalerUnmarshaler, cur *lrdd.Row) (next MarshalerUnmarshaler, err error)
+}
+
+type MarshalerUnmarshaler interface {
```

Comment:
IMHO, `MarshalUnmarshaler` is the more common naming pattern in Go.
```suggestion
type MarshalUnmarshaler interface {
```

Reply:
Done. 499d5c14818f25504449ab51adff6601b4445a6a